### PR TITLE
Add selenium fixtures

### PR DESCRIPTION
**Pull request by cbrenner04**

### CODE REVIEW

**Comment by cbrenner04** [comment]
@eschlange  do you know what's going on with the Hakiri thing?

File: spec/selenium_fixtures/users.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<% (1..1).each do |i| %>
```
Comment (eschlange) [inline spec/selenium_fixtures/users.yml]: Is this necessary for a single instance?

**Comment by eschlange** [comment]
Looks good.

File: spec/selenium_fixtures/users.yml (hunk 1 above)
Comment (cbrenner04) [inline spec/selenium_fixtures/users.yml]: yeah. we think it's a bug with rails but basically if I iterate through any of the fixtures in a file, I need to iterate through them all or they'll blow up when I try to load them

Comment (eschlange) [inline spec/selenium_fixtures/users.yml]: Ah yeah, I remember you mentioning that.

**Pull request merged**